### ISSUE TITLE
cSpell.import is not respecting language setting

### ISSUE THREAD

**Issue by pcastellazzi** [streetsidesoftware/vscode-spell-checker#5042]
I want to spell check my files in both English and Spanish.

I installed the following extensions:

Installed extensions:
- Code Spell Checker
- Spanish - Code Spell Checker

I updated my `settings.json ` to reflect this:

```jsonc
// ...
"cSpell.language": "en,es",
```

All good, it works as expected.

I want to use `cspell-cli` too, so i moved my configuration to an external file.

`~/.config/cspell/cspell.json`
```json
{
    "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
    "version": "0.2",
    "language": "en,es"
}
```

I updated my `settings.json ` to reflect this:

```jsonc
// ...
"cSpell.import": [ "${env:HOME}/.config/cspell/cspell.json" ],
```

All words in spanish are now marked as errors.

I also tried a YAML file
```yaml
---
$schema: https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json
version: '0.2'
language: en,es

# intentional
[{]}]
```

This should trigger a syntax error and it does not, but relevant to my case, when the file is correct the language settings is still ignored.

**Comment by Jason3S** [comment]
@pcastellazzi,

Thank you.

I need to clarify a few thing before I can help.

Do you mean that Spanish words are being marked as incorrect when using the command line but work fine in VSCode?

The most likely cause is that the command line tool cannot use the dictionaries added by the extensions.

You can tell the spell checker to download the dictionary via a CDN:

**`~/.config/cspell/cspell.json`**

```json
{
    "import": ["https://cdn.jsdelivr.net/npm/@cspell/dict-es-es/cspell-ext.json"],
    "language": "en,es"
}
```

See also: [CSpell - Spanish Dictionary](https://github.com/streetsidesoftware/cspell-dicts/tree/main/dictionaries/es_ES)


The command line tool can show you the available dictionaries: `npx cspell dictionaries`.

<img width="857" height="387" alt="Image" src="https://github.com/user-attachments/assets/c76ccf90-bd55-4b52-9fc1-9bfa11a12aee" />

It can also show you where it located words: `npx cspell trace aqua`

<img width="864" height="406" alt="Image" src="https://github.com/user-attachments/assets/7ccd5894-9078-4d1f-9185-dbd4373c8003" />

In the example above, it found the word in the Spanish dictionary, but since I didn't have my language set to `es` it is not enabled by default.

**Comment by pcastellazzi** [comment]
Hello,

Thank you for your time @Jason3S. Sorry about the confusion. I meant cSpell is not working on Visual Studio.

My current Visual Studio Code:
```plain
Version: 1.108.2
Commit: c9d77990917f3102ada88be140d28b038d1dd7c7
Date: 2026-01-21T13:52:09.270Z
Electron: 39.2.7
ElectronBuildId: 12953945
Chromium: 142.0.7444.235
Node.js: 22.21.1
V8: 14.2.231.21-electron.0
OS: Linux x64 6.17.13+deb13-amd64
```

In this screenshot you can see a Visual Studio Code running the cSpell extension correctly. 

<img width="2400" height="1600" alt="Image" src="https://github.com/user-attachments/assets/05c72a46-d812-4f2c-a9a1-28bce4c89a5c" />

In this screenshot you can see it fail.

<img width="2400" height="1600" alt="Image" src="https://github.com/user-attachments/assets/4526d1df-8528-486d-aa16-562edd077290" />


The only difference between the two is where the configuration is saved. In the first one, the `language` property is set with the Visual Studio Code settings, in the second one the configuration is stored at `~/.config/cspell/cspell.json` and imported.
